Exiled.Events/Patches/Events/Player/Transmitting.cs

```diff
@@ -45,9 +47,6 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     // hub
                     new CodeInstruction(OpCodes.Ldarg_0).MoveLabelsFrom(newInstructions[index]),
```

Comment:
```suggestion
                    // this
                    new CodeInstruction(OpCodes.Ldarg_0).MoveLabelsFrom(newInstructions[index]),
```

Reply:
Ignore the comments in my drafts.., I will fix, my concern is logic :( - I have <2 hours of sleep again 🙏

Reply:
😭